instrumentation/jedis/jedis-3.0/src/main/java/io/opentelemetry/javaagent/instrumentation/jedis/v3_0/JedisInstrumentation.java

```diff
@@ -59,14 +63,10 @@ public JedisInstrumentation() {
     public static void onEnter(
         @Advice.This Connection connection,
         @Advice.Argument(0) ProtocolCommand command,
+        @Advice.Argument(1) byte[][] args,
         @Advice.Local("otelSpan") Span span,
         @Advice.Local("otelScope") Scope scope) {
-      int callDepth = CallDepthThreadLocalMap.incrementCallDepth(Connection.class);
```

Comment:
Why this is removed?

Reply:
It's no longer needed - I changed the method matcher so that exactly one method matches; previously there were 2 or 3 methods that got matched.